packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.tsx

```diff
@@ -33,6 +34,25 @@ const NO_HORIZONTAL_WRAP = 'data-no-horizontal-wrap';
 const LARGE_DISTANCE_FROM_CENTER = 999999999;
 const LARGE_NEGATIVE_DISTANCE_FROM_CENTER = -999999999;
 
+let focusZoneStyles: string;
+
+const focusZoneClass: string = 'ms-FocusZone';
+function getRootClass() {
```

Comment:
Please add a comment; but also, is it possible to only add the class if we're indeed in the parked focus case? That would limit regression risk here.

Reply:
@joschect The build is broken because snapshots. I think if you address this comment, the snapshots should be in better shape.